contracts/test/PoolClaim.t.sol

```diff
@@ -515,6 +555,69 @@ contract PoolTest is TestUtil {
         return newAmt == uint256(0) ? uint256(0) : newAmt.mul(totalClaim).div(totalNewAmt);
     }
 
+    function getFundedAmount(uint256 val) internal pure returns(uint256) {
+        return constrictToRange(val, 100 * USD,  val);
+    }
+
+    function getDL(Pool pool, Loan loan, DebtLockerFactory dlFactory) internal view returns(address) {
+        return pool.debtLockers(address(loan),   address(dlFactory));
+    }
+
+    function generateLoanAndDepositAmount(
+        uint256 apr,
+        uint256 index,
+        uint256 numPayments,
+        uint256 requestAmount,
+        uint256 collateralRatio
+    ) internal returns(uint256 depositAmt) {
+        uint256[5] memory specs = getFuzzedSpecs(apr, index, numPayments, requestAmount, collateralRatio);
+        address[3] memory calcs = [address(repaymentCalc), address(lateFeeCalc), address(premiumCalc)];
+
+        loan  = bob.createLoan(address(loanFactory), USDC, WETH, address(flFactory), address(clFactory), specs, calcs);
+        loan2 = ben.createLoan(address(loanFactory), USDC, WETH, address(flFactory), address(clFactory), specs, calcs);
+
+        depositAmt = constrictToRange(specs[3], 100 * USD, 1E10 * USD, true);  // Fund value should be between 100 USD - 1B USD
+        uint256 estimatedLLBalance = depositAmt + 3 * depositAmt + 6 * depositAmt;
+        if (estimatedLLBalance < specs[3] * 2) {
+            uint256 delta = 2 * specs[3] - estimatedLLBalance;
+            depositAmt = (estimatedLLBalance + delta / 10); 
+        }
+    }
+    
+    function getLoanFundedAmounts(uint256 beforeLLBalance, uint256 rounds, uint256 loan1FundedCount, uint256 loan2FundedCount) internal returns(uint256 , uint256[] memory) {
+        uint256 maxAmountPerFundLoan = beforeLLBalance / rounds;
+        uint256 totalFundedAmount    = 0;
+
+        uint256[] memory fundedCount     = new uint256[](2);
+        uint256[] memory fundedAmounts   = new uint256[](rounds);
+        uint256[] memory amtFundedToLoan = new uint256[](2);
+
+        fundedCount[0] = loan1FundedCount;
+        fundedCount[1] = loan2FundedCount;
+
+        for (uint256 i = 0; i < rounds; i++) {
+            fundedAmounts[i]   = getFundedAmount(maxAmountPerFundLoan);
+            totalFundedAmount += fundedAmounts[i];
+        }
```

Comment:
We still need to figure out a way to randomize the fundedAmounts so theyre different. Lets time box this, so if we cannot come up with a solution decently soon, we can just revert to using constant portions of `depositAmt`